<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutMeDialog</class>
 <widget class="QDialog" name="AboutMeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: rgb(102, 16, 53);
}</string>
  </property>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>530</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Jpan</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: rgb(191, 191, 191)
}</string>
   </property>
   <property name="text">
    <string>Keep your informations private.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="firstNameBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft PhagsPa</family>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(84, 13, 43);
color: rgb(209, 209, 209);
border: 4px solid '#1a1818';
letter-spacing: 1px;
border-radius: 10px;
}

QPushButton::hover {
background-color: rgb(84, 13, 43);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lastNameBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft PhagsPa</family>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(84, 13, 43);
color: rgb(209, 209, 209);
border: 4px solid '#1a1818';
letter-spacing: 1px;
border-radius: 10px;
}

QPushButton::hover {
background-color: rgb(84, 13, 43);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="sexBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft PhagsPa</family>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(84, 13, 43);
color: rgb(209, 209, 209);
border: 4px solid '#1a1818';
letter-spacing: 1px;
border-radius: 10px;
}

QPushButton::hover {
background-color: rgb(84, 13, 43);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ageBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft PhagsPa</family>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(84, 13, 43);
color: rgb(209, 209, 209);
border: 4px solid '#1a1818';
letter-spacing: 1px;
border-radius: 10px;
}

QPushButton::hover {
background-color: rgb(84, 13, 43);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="salaryBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft PhagsPa</family>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(84, 13, 43);
color: rgb(209, 209, 209);
border: 4px solid '#1a1818';
letter-spacing: 1px;
border-radius: 10px;
}

QPushButton::hover {
background-color: rgb(84, 13, 43);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="emailBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft PhagsPa</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(84, 13, 43);
color: rgb(209, 209, 209);
border: 4px solid '#1a1818';
letter-spacing: 1px;
border-radius: 10px;
}

QPushButton::hover {
background-color: rgb(84, 13, 43);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="passwordBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>440</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft PhagsPa</family>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(84, 13, 43);
color: rgb(209, 209, 209);
border: 4px solid '#1a1818';
letter-spacing: 1px;
border-radius: 10px;
}

QPushButton::hover {
background-color: rgb(84, 13, 43);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
